src/app/view/components/login-as/login-as.ts

```diff
@@ -73,15 +71,19 @@ export class LoginAsComponent extends BaseComponent {
      *
      * @param event - The event with the query parameter to filter the users
      */
-    filterUsers(event): void {
-        this.filteredLoginAsUsersResults = this.userLists.
-        filter(user => user.fullName.toLowerCase().indexOf(event.query.toLowerCase()) >= 0)
-            .map(user => {
+    filterUsers(event?): void {
+        console.log('event', event);
+
+        let query = !event || !event.query ? null : event.query.toLowerCase();
```

Comment:
It's always easier to read when a condition start with a "positive" statement, way easier. So in this case:

```
let query = event || event.query ? event.query.toLowerCase() : null;
```

or I think you can do even:

```
let query = event.query.toLowerCase() || null;
```